src/Sparrow.Server/LowMemory/MemoryInformation.cs

```diff
@@ -257,8 +260,14 @@ struct IO_COUNTERS
             public UInt64 WriteTransferCount;
             public UInt64 OtherTransferCount;
         }
-        
-        
+
+        [DllImport("kernel32.dll", SetLastError = true)]
+        public static extern IntPtr GetCurrentProcess();
```

Comment:
This exists in `Win32MemoryQueryMethods`. You can take it from there.

Reply:
done